src/contraction/contractGraph_driver.cpp

```diff
@@ -94,7 +97,9 @@ void get_postgres_result(
     size_t sequence = 0;
 
     for (const auto id : modified_vertices) {
-        auto v = graph.get_V(id);  // vertex id
+        auto v = graph.get_V(id); // vertex id
+        int64_t o = graph.get_O(id); // order
+        int64_t m = graph.get_M(id); // metric
```

Comment:
`double` type?

Reply:
I finally stay on an `integer` everywhere in the code, since the present metric is `integer` and I do not know if we would need a `double` for other metrics.